modules/dkan/dkan_periodic_updates/dkan_periodic_updates.module

```diff
@@ -248,8 +254,10 @@ function dkan_periodic_updates_get_items_to_update($file) {
 function dkan_periodic_updates_execute_update($resources) {
   module_load_include('inc', 'dkan_datastore', 'resources/dkan_datastore_resource');
   if (!empty($resources)) {
-    foreach ($resources as $uuid => $file_url) {
+    foreach ($resources as $uuid => $resource) {
       $nids = entity_get_id_by_uuid('node', [$uuid]);
+      $file_url = $resource['file_url'];
+      $datastore = $resource['datastore'] === 'Y' ? TRUE : FALSE;
```

Comment:
I have a manifest with values: Y,n,n,blah in the 'import_to_datastore' column and none of the files are added to the datastore.